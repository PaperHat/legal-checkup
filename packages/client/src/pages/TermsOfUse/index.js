import React from 'react'

import Layout from '../../components/Layout'

export default function TermsOfUse () {
  return (
    <Layout>
      <p>
        This is the <strong>Terms of Use</strong> page
      </p>
    </Layout>
  )
}
