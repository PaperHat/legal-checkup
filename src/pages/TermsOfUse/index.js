import React from 'react'

import { Layout } from '../../components'

export default function TermsOfUse () {
  return <Layout>This is the Terms of Use page</Layout>
}
