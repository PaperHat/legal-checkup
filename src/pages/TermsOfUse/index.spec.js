import React from 'react'
import { shallow } from 'enzyme'

import TermsOfUse from '.'

describe('components:TermsOfUse', () => {
  it('renders the TermsOfUse page', () => {
    expect(toJson(shallow(<TermsOfUse />))).toMatchSnapshot()
  })
})
