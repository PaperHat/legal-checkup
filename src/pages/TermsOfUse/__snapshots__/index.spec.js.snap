// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:TermsOfUse  renders the TermsOfUse  page 1`] = `
<Layout>
  <p>
    This is the 
    <strong>
      Terms of Use
    </strong>
     page
  </p>
</Layout>
`;
